travis_ci_bootstrap_edm.sh

```diff
@@ -0,0 +1,15 @@
+if [[ ${TRAVIS_OS_NAME} == "osx" ]]
+then
+    # download and install EDM
+    wget https://package-data.enthought.com/edm/osx_x86_64/1.4/edm_1.4.0.pkg
```

Comment:
You should consider extracting the version number in a local variable, for ease of updating.